app/storage/encrypted_questionnaire_storage.py

```diff
@@ -1,14 +1,13 @@
 import hashlib
```

Comment:
I think we should drop the support for non-encrypted storage. Developers rarely used it previously, and now don't need to since we added the ability to dump the data via `/dump/answers` and `/dump/submission` in https://github.com/ONSdigital/eq-survey-runner/pull/1025